fapolicy_analyzer/ui/rules/rules_text_view.py

```diff
@@ -25,6 +27,14 @@ def __init__(self):
         self.__text_view = self.get_object("textView")
         self.__text_view.set_show_line_numbers(True)
 
-    def render_rules(self, rules: Sequence[Rule]):
-        text = "\n".join([r.text for r in rules])
+    def render_rules(self, rules_path: str):
+        text = ""
+        print(open)
```

Comment:
Does this still need to be here?

Reply:
Good catch. Left over from getting the tests running. It gone now.